docs/docs/tutorials/eth.md

```diff
@@ -0,0 +1,375 @@
+const { Web3 } = require('web3'); //  web3.js has native ESM builds and (`import Web3 from 'web3'`)
+
+// Set up a connection to the Ganache network
+const web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:7545'));
+
+// Log the current block number to the console
+web3.eth
+	.getBlockNumber()
+	.then(result => {
+		console.log('Current block number: ' + result);
+	})
+	.catch(error => {
+		console.error(error);
+	});
+```
+
+This code sets up a connection to the Ganache network and logs the current block number to the console.
+
+Run the following command to test the connection:
+
+```
+node index.js
+```
+
+If everything is working correctly, you should see the current block number logged to the console. However, if you got an error with the reason `connect ECONNREFUSED 127.0.0.1:7545` then double check that you are running Ganache locally on port `7545`.
+
+## Step 3: Interact with the Ethereum blockchain using web3.js
+
+In this step, we will use web3.js to interact with the Ganache network.
+
+In the first example, we are going to send a simple value transaction.
+Create a file named `transaction.ts` and fill it with the following code:
+
+```typescript
+const { Web3 } = require('web3'); //  web3.js has native ESM builds and (`import Web3 from 'web3'`)
+const fs = require('fs');
+const path = require('path');
+
+// Set up a connection to the Ethereum network
+const web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:7545'));
+web3.eth.Contract.handleRevert = true;
+
+// send a transaction
+const deployedAddressPath = path.join(__dirname, 'MyContractAddress.bin');
+const deployedAddress = fs.readFileSync(deployedAddressPath, 'utf8');
+
+// Read the bytecode from the file system
+const bytecodePath = path.join(__dirname, 'MyContractBytecode.bin');
+const bytecode = fs.readFileSync(bytecodePath, 'utf8');
+
+// Create a new contract object using the ABI and bytecode
+const abi = require('./MyContractAbi.json');
+const MyContract = new web3.eth.Contract(abi, deployedAddress);
+
+async function interact() {
+	//fetch all the available accounts
+	const accounts = await web3.eth.getAccounts();
+	console.log(accounts);
+
+	let balance1, balance2;
+	//The initial balances of the accounts should be 100 Eth (10^18 wei)
+	balance1 = await web3.eth.getBalance(accounts[0]);
+	balance2 = await web3.eth.getBalance(accounts[1]);
+
+	console.log(balance1, balance2);
+
+	//create a transaction sending 1 Ether from account 0 to account 1
+	const transaction = {
+		from: accounts[0],
+		to: accounts[1],
+		value: web3.utils.toWei('1', 'ether'), // value should be passed in wei. For easier use and to avoid mistakes we utilize the auxiliary `toWei` function.
+	};
+
+	//send the actual transaction
+	const transactionHash = await web3.eth.sendTransaction(transaction);
+	console.log('transactionHash', transactionHash);
+
+	balance1 = await web3.eth.getBalance(accounts[0]);
+	balance2 = await web3.eth.getBalance(accounts[1]);
```

Comment:
it would also be good to show the different return formats in one of these examples. what do u think?

Reply:
I use `ETH_DATA_FORMAT` in a later example (https://github.com/web3/web3.js/pull/6120/files#diff-c99987636f6f022dcffeaf9e5779f32ee485b7f5909785fdf97134bc58914ebbR249). I can add it in more places if you think it would be better.

Reply:
How do I start web 3 n I have matamask